TechDoc-Assistance-Program.md

```diff
@@ -0,0 +1,55 @@
+- Improve the chances of a good outcome from the Tech Doc Assistance Program
+
+For this reason, CNCF offers free training on documentation essentials for project contributors and maintainers. To get the most from the Technical Documentation Assistance Program, project contributors should do the training *before* engaging a documentation specialist to complete the documentation analysis.
+
+The training program consists of the following online courses. Anyone can sign up for and complete the courses at their own pace.
+
+1. [Open Source Technical Documentation Essentials (LFC111)](https://training.linuxfoundation.org/training/open-source-technical-documentation-essentials-lfc111/)
+1. [Creating Effective Documentation for Developers (LFC112)](https://training.linuxfoundation.org/training/creating-effective-documentation-for-developers-lfc112/)
+
+## Phase 1: Documentation Analysis
+
+A technical writer (on CNCF staff or on contract) analyzes the documentation. Based on the standards developed by the CNCF Tech Docs program, the writer:
+
+1. Estimates the maturity level of the documentation compared to the current or desired maturity level of the software project, based on a number of criteria in three categories, using a rubric developed by CNCF. The categories are:
+    1. Project documentation: The end-user documentation for the project's work product, typically (but not always) an application, API, protocol, or some other software product.
+    1. Contributor documentation: Documentation about the project, aimed at project contributors and describing procedures, infrastructure, and customs for doing project work. This includes artifacts that define procedures and governance; recruit, orient, and train project contributors; and name responsible parties (project leaders, often generically called *maintainers*).
+    1. Website: The technical infrastructure behind the documentation and the project's web presence, including website generation, versioning, SEO, analytics, and security.
+1. Collaborates with project leadership to identify user roles and objectives for software users. 
+1. Proposes changes, if necessary, to the organization and content of the documentation to close gaps with the proposed maturity level.
+1. Writes an implementation plan describing improvements that address the gaps identified in the analysis.
+
+## Phase 2: Backlog Creation
+
+Once a high-level improvement plan has been written and approved, the tech writer analyzes the proposed changes to create a backlog of actionable, individual writing assignments and other tasks to improve the documentation. These tasks should have two primary characteristics:
+- As much as possible, they should be independent of each other so they can be completed in any order by any combination of contributors; and
+- They should be time-constrained. If possible, large tasks should be broken down into smaller, independent tasks that require hours, not days, to complete.
```

Comment:
I'd say days can be ok, but not weeks, maybe something like:

```suggestion
- They should be time-constrained. If possible, large tasks should be broken down into smaller, independent tasks that require hours or days rather than weeks to complete.
```